docker/Dockerfile

```diff
@@ -27,6 +29,8 @@ RUN apt-get update -y && apt-get upgrade -y
 COPY --from=builder "/go/mx-chain-go/cmd/node" "/go/mx-chain-go/cmd/node/"
 COPY --from=builder "/lib/libwasmer_linux_amd64.so" "/lib/libwasmer_linux_amd64.so"
 COPY --from=builder "/lib/libvmexeccapi.so" "/lib/libvmexeccapi.so"
+COPY --from=builder "/lib/libwasmer_linux_arm64_shim.so" "/lib/libwasmer_linux_arm64_shim.so"
+COPY --from=builder "/lib/libvmexeccapi_arm.so" "/lib/libvmexeccapi_arm.so"
```

Comment:
Ideally, conditional copying based on the variable `TARGETPLATFORM`. Maybe do that in another PR?

Create a GH issue on this matter.

Reply:
Not applicable anymore :+1: